### PR TITLE
fix: Update list of tickets automatically on creation of new ticket

### LINKED ISSUE #1446: Tickets list is not updated after creating new ticket
**Describe the bug**
Whenever we create a new ticket, and return to the screen containing the list of the tickets, then we need to manually refresh the contents to see the newly added ticket. 

**To Reproduce**
Go to the Tickets section, create a new ticket and return to the tickets screen. The new ticket will nnot be present there. 

**Expected behavior**
The list should be updated. 

**Logs**
NA

**Screenshots**
ezgif com-video-to-gif 1

**Smartphone Info:**
|Device|         Redmi Note 4
|Android Version|  Nougat 7.1.1

**Additional context**

**Would you like to work on the issue?**

- [X] Yes
- [ ] No
- Other:

### PR DESCRIPTION
**Pull request by ShridharGoel**
Fixes #1446 

Changes: 

Now, list of tickets would get updated automatically on creation of new ticket.

### CODE REVIEW

**Comment by iamareebjamal** [comment]
Why not fetch from DB in onResume?

**Comment by ShridharGoel** [comment]
> Why not fetch from DB in onResume?

That's not working in this case.

File: app/src/main/java/com/eventyay/organizer/core/ticket/list/TicketsFragment.java [hunk 1]
```diff
@@ -97,6 +97,8 @@ public void onStop() {
     }
 
     public void openCreateTicketFragment() {
+        getPresenter().isNewTicketCreated = true;
```
Comment (iamareebjamal) [inline app/src/main/java/com/eventyay/organizer/core/ticket/list/TicketsFragment.java]: Why can't you set it from inside the presenter?

Comment (ShridharGoel) [reply app/src/main/java/com/eventyay/organizer/core/ticket/list/TicketsFragment.java]: I want to set it only when a new ticket is created. How to do it from inside the `TicketsPresenter`?

Comment (iamareebjamal) [reply app/src/main/java/com/eventyay/organizer/core/ticket/list/TicketsFragment.java]: Hmm, openCreateTicketFragment doesn't mean that ticket will be created

Comment (ShridharGoel) [reply app/src/main/java/com/eventyay/organizer/core/ticket/list/TicketsFragment.java]: Yes I know but then should we set it from `CreateTicketPresenter`?

Comment (iamareebjamal) [reply app/src/main/java/com/eventyay/organizer/core/ticket/list/TicketsFragment.java]: Try

Comment (ShridharGoel) [reply app/src/main/java/com/eventyay/organizer/core/ticket/list/TicketsFragment.java]: Updated.

File: app/src/main/java/com/eventyay/organizer/core/ticket/list/TicketsPresenter.java [hunk 2]
```diff
@@ -29,6 +29,8 @@
     private final TicketRepository ticketRepository;
     private final DatabaseChangeListener<Ticket> ticketChangeListener;
 
+    public static boolean isNewTicketCreated = false;
```
Comment (iamareebjamal) [inline app/src/main/java/com/eventyay/organizer/core/ticket/list/TicketsPresenter.java]: static variables are really bad

Comment (ShridharGoel) [reply app/src/main/java/com/eventyay/organizer/core/ticket/list/TicketsPresenter.java]: I had tried injecting `TicketsPresenter` but it didn't work as needed. Then, what should be done?

Comment (iamareebjamal) [reply app/src/main/java/com/eventyay/organizer/core/ticket/list/TicketsPresenter.java]: I'll see tomorrow

Comment (ShridharGoel) [reply app/src/main/java/com/eventyay/organizer/core/ticket/list/TicketsPresenter.java]: @iamareebjamal Please have a look.

Comment (ShridharGoel) [reply app/src/main/java/com/eventyay/organizer/core/ticket/list/TicketsPresenter.java]: @iamareebjamal Please suggest something for this.

Comment (iamareebjamal) [reply app/src/main/java/com/eventyay/organizer/core/ticket/list/TicketsPresenter.java]: Skip this issue for now

Comment (ShridharGoel) [reply app/src/main/java/com/eventyay/organizer/core/ticket/list/TicketsPresenter.java]: Okay

**Pull request closed without merging**